docs/feature-flagging/mutual_exclusion.md

```diff
@@ -0,0 +1,61 @@
+# Mutual exclusion (Layers)
+
+:::note
+Layers are currently in closed Beta
+:::
+
+There are situations when you want to run concurrent experiments on the same surface. Eppo offers Layers as a option to keep your experiments mutual exclusive.
+
+Research from Microsoft has shown that in practice [interaction effects are vanishingly rare](https://www.microsoft.com/en-us/research/group/experimentation-platform-exp/articles/a-b-interactions-a-call-to-relax/). Given this, we recommend only using mutual exclusion when the overlap of two new treatments critically degrades the user experience. For instance, a design team might want to run a homepage redesign test while a marketing team might want to experiment on the headline messaging on the same page. If the new layout removes the messaging altogether or makes it unreadable, these experiments should clearly not overlap.
```

Comment:
I would put this in a callout block (similar to the note for layers, although maybe use info or something like that)